src/screens/DiscoveryScreenV2/fragment/SearchFragment.js

```diff
@@ -15,7 +15,7 @@ const SearchFragment = () => (
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the background color of the container in `SearchFragment.js` might significantly alter the appearance of this screen. Ensure that this color change aligns with the overall design language and accessibility standards (contrast, readability) of your application.
```diff
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->